src/main/java/wooteco/subway/web/member/LoginMemberController.java

```diff
@@ -20,26 +24,44 @@ public LoginMemberController(MemberService memberService) {
     }
 
     @PostMapping("/oauth/token")
-    public ResponseEntity<TokenResponse> login(@RequestBody LoginRequest param) {
+    public ResponseEntity<TokenResponse> login(@Valid @RequestBody LoginRequest param) {
         String token = memberService.createToken(param);
         return ResponseEntity.ok().body(new TokenResponse(token, "bearer"));
     }
 
-    @PostMapping("/login")
-    public ResponseEntity login(@RequestParam Map<String, String> paramMap, HttpSession session) {
-        String email = paramMap.get("email");
-        String password = paramMap.get("password");
-        if (!memberService.loginWithForm(email, password)) {
-            throw new InvalidAuthenticationException("올바르지 않은 이메일과 비밀번호 입력");
-        }
+    @GetMapping("/me/bearer")
+    public ResponseEntity<MemberResponse> getMemberOfMineBasic(@LoginMember Member member) {
+        return ResponseEntity.ok().body(MemberResponse.of(member));
+    }
 
-        session.setAttribute("loginMemberEmail", email);
+    @PutMapping("/me")
+    public ResponseEntity<Void> updateMember(@LoginMember Member member, @RequestBody UpdateMemberRequest param) {
+        memberService.updateMember(member, param);
+        return ResponseEntity.ok().build();
+    }
 
+    @DeleteMapping("/me")
+    public ResponseEntity<Void> deleteMember(@LoginMember Member member) {
+        memberService.deleteMember(member);
         return ResponseEntity.ok().build();
```

Comment:
```suggestion
        return ResponseEntity.noContent().build();
```

아래 removeFavorite과 같이 수정해보면 어떨까요?

Reply:
수정했습니다.